content/material_and_questions.md

```diff
@@ -13,23 +13,31 @@
 | 2026-2027 | John                                          | Narrative |
 | 2027-2028 | Hebrews, 1 Peter, 2 Peter                     | Epistle   |
 
-## Question Writing
+## Material
+: The biblical content that is the basis for competition for the quiz year. This includes Book Name, Chapter Number, and Verse Number. This does not include any specifically excluded material.
```

Comment:
These new paragraphs begin with a `:`, but they aren't strictly definitions in the Markdown sense. (Either they have to be preceded with a term or not be definitions.) I'd remove the headers and turn these into terms, unless any are multiple paragraphs, then just remove the `:`.

Reply:
Makes sense, I think.  There's not really any "Definitions" header I can put my content underneath, since it's kinda interspersed.